docs/shard.yml.adoc

```diff
@@ -204,8 +204,13 @@ libraries:
 ----
 
 *license*::
-  An http://opensource.org/[OSI license] name or an URL to a license file
+An https://spdx.org/licenses/[SPDX license identifier] or an URL to a license file
 (string, recommended).
++
+The OSI publishes https://opensource.org/licenses-old/category[a list] of open
+source licenses and their corresponding SPDX identifier.
++
+Examples: _Apache-2.0_, _MIT_, _GPL-3.0-or-later_.
```

Comment:
**suggestion:** Add an example for a URL pointing to a custom license file that's not covered by a SPDX identifier (it can be an `example.com` URL).